dashboard/app/models/user_level.rb

```diff
@@ -98,6 +98,11 @@ def handle_unsubmit
     if submitted_changed? from: true, to: false
       self.best_result = ActivityConstants::UNSUBMITTED_RESULT
     end
+
+    # Destroy any existing peer reviews
+    if level.try(:peer_reviewable?)
+      PeerReview.where(submitter: user.id, reviewer: nil, level: level).destroy_all
```

Comment:
Is reviewer always going to be nil, or could there be existing peer reviews that have a reviewer set?